migrations/20210817002856-description.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+module.exports = {
+  up: function (queryInterface, Sequelize) {
+    // logic for transforming into the new state
+    return queryInterface.addColumn('Resources', 'description', Sequelize.STRING);
```

Comment:
Since we're collecting what might be more than one line of text, this should also be a TEXT column to match how we've defined it in the model...